sunpy/net/tests/test_helioviewer.py

```diff
@@ -74,6 +74,21 @@ def test_download_jp2(self, client):
         assert "2011_01_11__22_39_00_000__SOHO_MDI_MDI_continuum.jp2" in filepath
         os.remove(filepath)
 
+    def test_get_jp2_header(self, client):
+        """
+        Tests getJP2Header API method
+        """
+        header1 = client.get_jp2_header('1994/01/01', observatory='SOHO',
+                                              instrument='EIT', measurement='304')
+        header2 = client.get_jp2_header('1994/01/01', jp2_id = 1795504)
+        keys_header1 = header1.keys()
+        keys_header2 = header2.keys()
+        assert cmp(header1 == header2)
+        assert len(header1) == 2
+        assert len(header2) == 2
+        assert 'fits' and 'helioviewer' in keys_header1
+        assert 'fits' and 'helioviewer' in keys_header2        
```

Comment:
We can probably shorten the lines. 

```
        assert cmp(header1 == header2)
        assert len(header1) == len(header2) == 2
        assert 'fits' and 'helioviewer' in header1.keys()
        assert 'fits' and 'helioviewer' in header2.keys()  

 ```
I also wonder if the last two lines could be made into one.